app.js

```diff
@@ -0,0 +1,64 @@
+const express = require("express");
+const cors = require("cors");
+const app = express();
+const bodyParser = require("body-parser");
+const port = 3000;
+const School = require("./School");
+const Student = require("./Student");
+
+app.use(cors());
+app.use(bodyParser.urlencoded({ extended: false }));
+app.use(bodyParser.json());
+
+let mySchool = new School();
+
+// Create a Class w/ Course Name and Teacher
+app.post("/class", (req, res) => {
```

Comment:
Before you add new class you need to check if the class exists already or not. You can do that using middleware functions.